src/wireviz/DataClasses.py

```diff
@@ -18,6 +18,10 @@ class Connector:
     type: Optional[str] = None
     subtype: Optional[str] = None
     pincount: Optional[int] = None
+    image: Optional[str] = None
+    image_scale: Optional[str] = "FALSE"
```

Comment:
why not make this a boolean?

Reply:
Because it has several valid values as described in https://github.com/formatc1702/WireViz/issues/27#issuecomment-670141605.

Reply:
I see. In the YAML, maybe we can use lowercase strings (`false`, `true`, `width`) to keep it consistent, and the helper function can take care to convert `false` and `true` (which the YAML parser will interpret as boolean) to strings, and make them uppercase before including in the HTML.

Reply:
Graphviz is not case sensitive. We can use upper, lower, or capital case. The Graphviz doc uses uppercase on all tags and attributes, but lowercase seems to be the standard in this project. The YAML parser interpretes both `true` and `TRUE` as boolean, and the f-string result is `True` in both cases, but Graphviz don't care. I can change to lowercase in comments and constant strings, but do we also need `str(value).lower()` in the f-string when Graphviz don't really care?

Reply:
[As you wish 😎](https://www.youtube.com/watch?v=CxK_nA2iVXw)